docs/user-guide/global-configurations/authorization/sso/google.md

```diff
@@ -0,0 +1,38 @@
+# Google
+
+## Sample Configuration
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/google.jpg)
+
+---
+
+## Values You Would Require at SSO Provider
+
+Devtron provides a sample configuration out of the box. There are some values that you need to either get from your SSO provider or give to your SSO provider.
+
+### Values to Fetch
+
+* clientID
+
+* clientSecret
+
+    ![Fetching Client ID and Secret](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/secret/google-id-secret.jpg)
+
+
+### Values to Provide
+
+* redirectURI (provided in SSO Login Services by Devtron)
```

Comment:
Hosted domain and its use is missing, also the attached screenshot has a definite redirect url, hope its not confusing, how to get a redirect URL? that can also be added

Reply:
Maybe highlight the redirect URL which we show above as a person can use that directly

Reply:
Hosted domain part is covered in DexIDP documentation whose link we have already provided in our UI. It's practically not possible to cover every single configuration key-value pairs in our doc. Initial intention was to include the process of fetching Client ID and Secret correctly from the SSO provider (which is not covered by DexIDP).

Reply:
> Maybe highlight the redirect URL which we show above as a person can use that directly

Done...added the snapshots